source/tyr/tyr/tasks.py

```diff
@@ -130,9 +130,11 @@ def import_data(files, instance, backup_file, async=True, reload=True, custom_ou
         if reload:
             actions.append(reload_data.si(instance_config, job.id))
 
-        for dataset in job.data_sets:
-            if dataset.family_type == 'pt':
-                actions.extend(send_to_mimir(instance, dataset.name))
+        # Test parameters for instance for send_to_mimir
+        if instance.import_ntfs_in_mimir or instance.import_stops_in_mimir:
```

Comment:
To me it would be better to group it in send_to_mimir with all the tests on those params (just return empty actions in that case).
The code would be less scattered, easier to maintain.